website/docs/configuration/functions/can.html.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: "functions"
+page_title: "can - Functions - Configuration Language"
+sidebar_current: "docs-funcs-conversion-can"
+description: |-
+  The can function tries to evaluate an expression given as an argument and
+  indicates whether the evaluation succeeded.
+---
+
+# `can` Function
+
+-> **Note:** This page is about Terraform 0.12 and later. For Terraform 0.11 and
+earlier, see
+[0.11 Configuration Language: Interpolation Syntax](../../configuration-0-11/interpolation.html).
+
+`can` evaluates the given expression and returns a boolean value indicating
+whether the expression produced a result without any errors.
+
+This is a special function that is able to catch errors produced when evaluating
+its argument. For most situations where you could use `can` it's better to use
+[`try`](./try.html) instead, because it allows for more concise definition of
+fallback values for failing expressions.
+
+The primary purpose of `can` is to turn an error condition into a boolean
```

Comment:
What do you think about explicitly (briefly) mentioning that the validation is experimental here? I'm concerned about users finding this page and not clicking through to the variables documentation.

Reply:
I was feeling a little reluctant to sprinkle the experimental bits all over because eventually we're going to need to clean them back up again, and Terraform itself will quickly tell the user that the feature is experimental if they try to use it. With that said, a key goal of this PR is to rehearse the experimental feature lifecycle, and the degree to which we let it "invade" our documentation is likely to be one of those things that we'll adjust over time with experience, so maybe I could add it in here now in the interests of making this a more extreme case to learn from. (Now that we've discussed it specifically together we're less likely to forget about it, of course, but still...)

Reply:
I added a note to the example below this paragraph. I guess we'll refer back to this PR when it comes time to un-experiment-ize this and then hopefully this conversation will be our prompt to remove the note! (This'll be a good first test of our experiment lifecycle procedures. :wink: )